test/vaadin-list-mixin.html

```diff
@@ -281,6 +281,47 @@
 
         beforeEach(() => list._focus(0));
 
+        describe('RTL mode', () => {
+
+          beforeEach(() => list.setAttribute('dir', 'rtl'));
+
+          it('should move focus to next element on "arrow-down" keydown', () => {
+            arrowDown(list);
+            expect(list.items[1].focused).to.be.true;
+          });
+
+          it('should move focus to prev element on "arrow-up" keydown', () => {
+            arrowDown(list);
+            arrowUp(list);
+            expect(list.items[0].focused).to.be.true;
+          });
+
+          it('should move focus to prev element on "arrow-left" keydown', () => {
+            list.orientation = 'horizontal';
+            arrowLeft(list);
+            expect(list.items[1].focused).to.be.true;
+          });
+
+          it('should move focus to next element on "arrow-right" keydown', () => {
+            list.orientation = 'horizontal';
+            arrowLeft(list);
+            arrowRight(list);
+            expect(list.items[0].focused).to.be.true;
+          });
+
+          it('should move focus to last element on "home" keydown', () => {
+            list._focus(3);
```

Comment:
This line should be moved from `home` test to `end` test below (to ensure End moves to first item).

Reply:
Good catch! Updated.